App.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+var ShallowRenderer = require('react-test-renderer/shallow');
+
+import App from './App';
+
+describe('<App />', () => {
+    it('has 1 child', () => {
+        const tree = renderer.create(<App />).toJSON();
+        expect(tree.children.length).toBe(1);
+    });
+    it('renders correctly', () => {
+        const tree = renderer.create(<App />).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
```

Comment:
Wahey! Tests from the first PR 👍

How should I run these? There's a `test` script in `package.json` but both tests fail and the file pattern doesn't actually match this file.

We'll probably want to focus the snapshot tests on smaller components as the project progresses. They're great for ensuring we're not affecting the output of a component as we refactor - but as almost any change will modify the application the snapshots will most likely be rebuilt each time.

Reply:
I just ran npm run test  or install jest globally and run jest - they seem to be failing since updating App with react navigation.
I also didn't update the snapshot but updating locally seems to return null. Still looking into this, not too familiar with jest.

Reply:
Cool cool, maybe this would be a good first thing to pair on to make sure we both understand how testing will work in the app

Reply:
Updates:

Fixed previous test errors by updating snapshots and adding mock for 'PersistGate' which was causing App to return null initially - https://github.com/rt2zz/redux-persist/issues/822#issuecomment-388165131

Added additional unit tests for LogButton component and countReducer reducer.

Modified test script in package.json: running 'npm run test <test file name>' should run individual test file or just 'npm run test' to run all.